jekyll/_posts/2022-02-04-nim-devroom-fosdem-2022.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Nim Devroom at FOSDEM 2022"
+author: pietroppeter, pmunch, dom96
+excerpt: "Tomorrow (Sat, Feb 5th) there is a Nim Devroom at FOSDEM 2022"
+---
+
+## Nim Devroom at FOSDEM 2022
+
+We are happy to announce that all preparations for the Nim Devroom at FOSDEM 2022 are now complete and
+we are looking forward to 8 excellent talks with more than 5 hours of Nim-related content. The conference is online this year
+and the talks will stream live tomorrow on the FOSDEM website. This is the final schedule for the Nim Devroom, which will be
+held tomorrow (Saturday, February 5th) from 12:30 to 17:40: https://fosdem.org/2022/schedule/track/nim_programming_language/
+
+**Note that all times are Brussels' times (CET).**
```

Comment:
Nitpick (you'd write "London time" not "London's time" so the `'` isn't necessary)

```suggestion
**Note that all times are Brussels time (CET).**
```

Reply:
Alternatively, maybe something like:

> **Note that all times are Central European Time (UTC+1)**.

which helps a reader who isn't familiar with CET, or is unsure about how CET interacts with daylight savings time.

Reply:
I will go with the suggestion by @ee7